tools/pv_tree_generator/constants.py

```diff
@@ -0,0 +1,261 @@
+# Lint as: python3
+
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""constants used when creating the pv tree."""
+
+import collections
+
+API_KEY = "AIzaSyCI-BiRt4rsByNGju_3-rQ90mqfD5XN5S4"
```

Comment:
should download the api key as instructed in  https://github.com/datacommonsorg/website/blob/master/README.md#download-the-config-file-that-contains-api-key.

And import the api key from the configmodule.py

Reply:
I don't have access to the configmodule.py

Reply:
done